getter/get.py

```diff
@@ -225,6 +228,15 @@ def fetch_and_convert(args, dataset):
             data_acceptable_license.append(dataset)
 
 
+def file_cache_schema():
+    tmp_dir = tempfile.mkdtemp()
+    schema_path = os.path.join(tmp_dir, '360-giving-schema.json')
+    schema = requests.get('https://raw.githubusercontent.com/ThreeSixtyGiving/standard/master/schema/360-giving-schema.json')
```

Comment:
Note to self, or reviewer: Perhaps need a try except block in here to terminate if the download fails?